.github/dependabot.yml

```diff
@@ -10,6 +10,53 @@ updates:
     directory: "/"
     # The maximum number of pull requests for version updates
     open-pull-requests-limit: 10
+    groups:
+      radix-ui:
+        patterns:
+          - "@radix-ui/*"
+      graphql:
+        patterns:
+          - "graphql"
+          - "@graphql-codegen/*"
+          - "@0no-co/graphqlsp"
+      eslint:
+        patterns:
+          - "eslint"
+          - "eslint-*"
+          - "@eslint/*"
+      typescript:
+        patterns:
+          - "typescript"
+          - "tsx"
+          - "@types/*"
+      prettier:
+        patterns:
+          - "prettier"
+          - "@prettier/*"
```

Comment:
Please reconsider having this group at all.

As far as I can tell there are no `@prettier/*` packages in `package.json` at the moment.